### ISSUE TITLE
GitHub version differs from PyPI version

### ISSUE THREAD

**Issue by MelanieU** [uber/causalml#138]
I'm getting an error when pip installing causalml. However, in GitHub the code is edited which wouldn't throw me an error.
PyPI:
```
/local_disk0/pythonVirtualEnvDirs/virtualEnv-b92cbbcc-5bd9-4bd7-835c-fc4bb9fb958b/lib/python3.7/site-packages/causalml/inference/tree/models.py in growDecisionTreeFrom(self, X, treatment, y, evaluationFunction, max_depth, min_samples_leaf, depth, min_samples_treatment, n_reg, parentNodeSummary)
    918 
    919         for col in list(np.random.choice(a=range(columnCount), size=max_features, replace=False)):
--> 920             columnValues = X[col]
```
GitHub:
```
--> 920             columnValues = X[:,col]
```

**Comment by jeongyoonlee** [comment]
The fix will be released in 0.8.0 by the end of this month.

**Comment by jeongyoonlee** [comment]
0.7.0 release on 2/28/2020 resolves this issue.
